src/store/index.ts

```diff
@@ -1,54 +1,85 @@
 import { createStore, AnyAction } from 'redux';
 import { composeWithDevTools } from 'redux-devtools-extension';
 
-// Action types - is just a constant. MUST have a unique value.
-const START_LOADING = 'START_LOADING';
-const FINISH_LOADING = 'FINISH_LOADING';
-
-// Action creators - a function returning an action object
-export const startLoading = () => ({ type: START_LOADING });
-export const finishLoading = (message = 'No message') => ({
-  type: FINISH_LOADING,
-  message,
+const TODOS_LOADING = 'TODOS_LOADING';
+const USER_LOADING = 'USER_LOADING';
+const SELECT_USER_ID = 'SELECT_USER_ID';
+const DELETE_TODO = 'DELETE_TODO';
```

Comment:
Should export these consts and use them for type on call dispatch

`dispatch({ type: USER_LOADING, ...);`